keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -184,7 +189,18 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
         |      "name": "prod",
         |      "constraints": [
         |        {
+        |          "type": "depends-on",
         |          "environment": "test"
+        |        },
+        |        {
+        |          "type": "time-window",
+        |          "windows": [
+        |            {
+        |              "hours": "6-18",
+        |              "days": "Monday-Friday",
```

Comment:
With this format can I specify something like "Monday, Wednesday and Friday"?

Reply:
Or is that just 3 separate windows?

Reply:
"and" isn't valid but you could specify "Monday, Wednesday, Friday"

Reply:
you can do a list of ranges/individual days: https://github.com/spinnaker/keel/pull/586/files#diff-cb20a8e71cc1d6a198f43860b58d5696R149

Reply:
@asher jinx.